docs/index.md

```diff
@@ -2,14 +2,14 @@
 title: Home
 ---
 
-# ![Ragna logo](assets/brand/logo-lockup-horizontal/logo-lockup-horizontal.png)
-
-Ragna is an open source RAG orchestration framework.
+# Ragna — Open source RAG orchestration framework
```

Comment:
The logo takes up a lot of space, and using a smaller version of the logo doesn't look that good, hence this update.